governance/mission-values.md

```diff
@@ -16,29 +15,74 @@ Science is now data-driven. And as such, data science supports scientists
 processing the data that they need to address critical questions.
 
 We value open science workflows because they allow scientists to build upon each
-others work rather than recreating already developed workflows. Building on
-top of existing workflows supports science developing and evolving at a quicker pace.
+other's work rather than recreating already developed workflows. Building on
+top of existing workflows accelerates science as it prevents scientists from needing to rebuild existing workflows from the ground up.
```

Comment:
These two sentences are pretty repetitive. What if the second one said something like "This accelerates science by allowing researchers to focus on novel questions and approaches." Feel free to wordsmith!